### PR TITLE
Swap out select2 for selectWoo to avoid incompatibility with older versions

### LINKED ISSUE #4541: Select2 version incompatibility with The7 theme
relationship list field does not show selected items. see attached pods  and JS screenshot
capture1
capture2

### PR DESCRIPTION
**Pull request by pglewis**
Fixes #4541

### CODE REVIEW

**Comment by pglewis** [comment]
Ready for testing but hold on merging for a bit.  This was a quick refactor and has not yet been tested rigorously.

File: classes/PodsInit.php [hunk 1]
```diff
@@ -293,8 +293,8 @@ public function register_assets() {
 			wp_register_style( 'jquery-ui-timepicker', PODS_URL . 'ui/js/timepicker/jquery-ui-timepicker-addon.min.css', array(), '1.6.3' );
 		}
 
-		wp_register_script( 'pods-select2', PODS_URL . 'ui/js/select2/select2.min.js', array( 'jquery', 'pods-i18n' ), '4.0.3' );
-		wp_register_style( 'pods-select2', PODS_URL . 'ui/js/select2/select2.min.css', array(), '4.0.3' );
+		wp_register_script( 'pods-select2', PODS_URL . 'ui/js/selectWoo/selectWoo.min.js', array( 'jquery', 'pods-i18n' ), '4.0.3' );
+		wp_register_style( 'pods-select2', PODS_URL . 'ui/js/selectWoo/selectWoo.min.css', array(), '4.0.3' );
```
Comment (jamesgol) [inline classes/PodsInit.php]: I know it wasn't there before, but this might be a good time to add a SCRIPT_DEBUG check to not use the minified versions

**Comment by jamesgol** [comment]
Initial testing looks good, I'll see if I can find some more time for more testing

**Comment by pglewis** [comment]
I would like to get a look-over from one of our CSS gurus if they have the time for sanity checks.

**Comment by pglewis** [comment]
Just holding on #4550 to get merged and merge back up here so tests pass.

**Pull request merged**